CAIPs/caip-104.md

```diff
@@ -108,9 +108,9 @@ name and github username or email>
 
 ` * requires:` <CAIP number(s)>
 
-` * replaces:` <CAIP number(s)>
+` * replaces:` <CAIP number(s) | namespace reference(s)>
```

Comment:
bit confused there - will a CAIP replace a namespace reference? I thought it is the other way around?

Reply:
this is the template for new namespaces-- so it can replace a CAIP or another namespace :D 

but you've inspired me to include an example for clarity